components/tidb_query_vec_expr/src/impl_time.rs

```diff
@@ -148,6 +148,22 @@ pub fn from_days(ctx: &mut EvalContext, arg: Option<&Int>) -> Result<Option<Time
     })
 }
 
+#[rpn_fn]
+#[inline]
+pub fn month_name(tmp: Option<&DateTime>) -> Result<Option<Bytes>> {
```

Comment:
The copr framkework has been changed, please remove the `Option` here, take [this PR](https://github.com/tikv/tikv/pull/8728) as ref.

BTW, please considering use `arg` instead of `tmp` here.